src/bokeh/core/property/pd.py

```diff
@@ -23,7 +23,11 @@
 # Standard library imports
 from typing import TYPE_CHECKING, Any
 
+# External imports
+import narwhals.stable.v1 as nw
```

Comment:
How expensive are these imports? We deferred all pandas imports and banned them at the top level, even though it is a hard dependency, because they are heavy and not all users use them. Unless this is a very fast import we might want to do the same with these.

Reply:
on my laptop (including pandas and logging for scale):
```
(scratch) marcogorelli@DESKTOP-U8OKFP3:~/scratch$ time python -c 'import pandas'

real    0m0.437s
user    0m1.981s
sys     0m0.037s
(scratch) marcogorelli@DESKTOP-U8OKFP3:~/scratch$ time python -c 'import narwhals.stable.v1 as nw'

real    0m0.033s
user    0m0.030s
sys     0m0.000s

(scratch) marcogorelli@DESKTOP-U8OKFP3:~/scratch$ time python -c 'import logging'

real    0m0.022s
user    0m0.011s
sys     0m0.010s
```

So, >10x faster than pandas, but a bit slower than some stdlib modules...I can move it out of the top level anyway, thanks for the suggestion (and for your review! will address the other points)